examples/sso-to-pingdirectory-from-pingone/README.md

```diff
@@ -0,0 +1,43 @@
+
+## Setting Connection Details
+The following environment variables should be set prior to running the example.  For more information, see the registry documentation for each provider.
+
+| Variable name                                   | Schema Documentation                                                                                                                            | Required/Optional |
+|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `PINGDIRECTORY_PROVIDER_USERNAME`               | [PingDirectory - username](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#username)                             | Required          |
+| `PINGDIRECTORY_PROVIDER_PASSWORD`               | [PingDirectory - password](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#password)                             | Required          |
+| `PINGDIRECTORY_PROVIDER_HTTPS_HOST`             | [PingDirectory - http_host](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#https_host)                          | Required          |
+| `PINGDIRECTORY_PROVIDER_INSECURE_TRUST_ALL_TLS` | [PingDirectory - insecure_trust_all_tls](https://registry.terraform.io/providers/pingidentity/pingdirectory/latest/docs#insecure_trust_all_tls) | Optional          |
+| `PINGONE_CLIENT_ID`                             | [PingOne - client_id](https://registry.terraform.io/providers/pingidentity/pingone/latest/docs#client_id)                                       | Required          |
+| `PINGONE_CLIENT_SECRET`                         | [PingOne - client_secret](https://registry.terraform.io/providers/pingidentity/pingone/latest/docs#client_secret)                               | Required          |
+| `PINGONE_ENVIRONMENT_ID`                        | [PingOne - environment_id](https://registry.terraform.io/providers/pingidentity/pingone/latest/docs#environment_id)                             | Required          |
+| `PINGONE_REGION`                                | [PingOne - region](https://registry.terraform.io/providers/pingidentity/pingone/latest/docs#region)                                             | Required          |
+
+
+## Setting Variables
+The following variables can be set prior to running the example:
+
+| Variable name                    | Required | Data Type | Default Value            |
+|----------------------------------|----------|-----------|--------------------------|
+| `pingdirectory_console_base_url` | No       | String    | `https://localhost:8443` |
+| `pingdirectory_ldap_host`        | No       | String    | *no default*             |
+| `pingdirectory_ldap_port`        | No       | Number    | *no default*             |
```

Comment:
If you want to provide defaults here, `1636` would be a good default for the port (it's the default LDAPS port for PD). And if we are using localhost as the default for the console, we could use localhost as the default for the PD server.